### PR TITLE
Automate Qase: 226, 228, 295, 297, 298,238, 299

### PR DESCRIPTION
**Pull request by valaparthvi**
### Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #

### Checklist:
- [ ] Squashed commits into logical changes
- [ ] Documentation
- [ ] GitHub Actions (if applicable)

### CODE REVIEW

File: hosted/aks/helper/helper_cluster.go [hunk 1]
```diff
@@ -564,6 +580,26 @@ func DeleteAKSClusteronAzure(clusterName string) error {
...
 }
 
+// ShowAKSStatusOnAzure returns the status of AKS based on the passed query
+// For e.g. '.currentKubernetesVersion' will only return the current kubernetes version of the cluster
+func ShowAKSStatusOnAzure(clusterName, resourceGroup, query string) (out string, err error) {
+	fmt.Println("Showing AKS cluster ...")
+	args := []string{"az", "aks", "show", "--subscription", subscriptionID, "--name", clusterName, "--resource-group", resourceGroup, "|", "jq", "-r", query}
+	cmd := strings.Join(args, " ")
+	fmt.Printf("Running command: %s\n", args)
+	out, err = proc.RunW("bash", "-c", cmd)
```
Comment (thehejik) [inline hosted/aks/helper/helper_cluster.go:590]: shouldn't we use the same approach as before? like:
```suggestion
	out, err = proc.RunW("az", args...)
```
(args/cmd would need to be adapted as well)

File: hosted/aks/helper/helper_cluster.go [hunk 2]
```diff
@@ -564,6 +580,26 @@ func DeleteAKSClusteronAzure(clusterName string) error {
...
+	return strings.TrimSpace(out), err
+}
+
+func ShowAKSNodePoolOnAzure(nodepoolName, clusterName, resourceGroup, query string) (out string, err error) {
+	fmt.Println("Showing AKS node pool ...")
+	args := []string{"az", "aks", "nodepool", "show", "--subscription", subscriptionID, "--name", nodepoolName, "--cluster-name", clusterName, "--resource-group", resourceGroup, "|", "jq", "-r", query}
+	cmd := strings.Join(args, " ")
+	fmt.Printf("Running command: %s\n", args)
+	out, err = proc.RunW("bash", "-c", cmd)
```
Comment (thehejik) [inline hosted/aks/helper/helper_cluster.go:599]: dtto

File: hosted/aks/helper/helper_cluster.go (hunk 1 above)
Comment (valaparthvi) [reply hosted/aks/helper/helper_cluster.go:590]: We are using `jq -r` to parse JSON with a specific query, so I had used `bash`, but I noticed `az` also has a `--query` parameter that serves the same purpose.  I will use that for both places. Thanks for the review.

Comment (thehejik) [reply hosted/aks/helper/helper_cluster.go]: Ah, right, I did not notice the jq stuff but indeed it should be doable over `--query`

**Pull request closed without merging**